src/main/java/language/Language.java

```diff
@@ -0,0 +1,34 @@
+package language;
+
+public interface Language {
```

Comment:
👍

Reply:
This interface looks quite large.

Reply:
An interface shouldn't be too large? Good to know